lib/grape/endpoint.rb

```diff
@@ -232,6 +232,7 @@ def run(env)
 
       route_setting(:saved_validations).each do |validator|
         begin
+          validator.context = Hashie::Mash.new(request: request, cookies: cookies)
```

Comment:
Assigning anything on a validator inside a request seems bad. It now holds onto the request, and the validator is attached to the endpoint. So when is request going to be GCed?